### PR TITLE
Deploy to production

### PR DESCRIPTION
**Pull request by azophy**
## CHANGELOG
- Groupware model: fix struktural filtering in get_users()
- Add traceback info in bot.process_error() when IS_DEBUG is True
- Fix docker build error
- Add validation in set_alias() method to avoid alias conflict 
- Set default username for checkin & checkout command

## Evidence

- title: Deploy bot telegram to production
- project: Digiteam
- participants: @azophy @firmanJS

### CODE REVIEW

**Review by firmanJS — APPROVED** [review]

File: models/bot.py [hunk 1]
```diff
@@ -213,6 +213,9 @@ def send_result(result):
 def process_error(telegram_item, e):
     """ process (and may be notify) error encountered """
     msg = str(e)
+    if os.getenv('IS_DEBUG') and e.__traceback__ :
+        msg += "\n"
+        msg += ''.join(traceback.format_exception(type(e), e, e.__traceback__))
     print('error:', msg)
```
Comment (firmanJS) [inline models/bot.py:219]: @azophy ada print yang belum di hapus

File: models/user.py [hunk 2]
```diff
@@ -50,6 +50,9 @@ def set_alias(username, new_alias):
 
     print('set alias for : username', username, 'new_alias', new_alias)
```
Comment (firmanJS) [inline models/user.py:51]: ini juga

File: models/bot.py (hunk 1 above)
Comment (azophy) [reply models/bot.py:219]: hoo, oke2 kang. aku bersihin dulu

File: models/user.py (hunk 2 above)
Comment (azophy) [reply models/user.py:51]: sip2

**Pull request merged**